### DISCUSSION TITLE
Добавить Ford Police Interceptor Utility.

### DISCUSSION THREAD

**Discussion by MarkTwel** [octothorp-team/public-dobrograd#913] [Предложения]
**Привет!👋🏿**  **Прочитай полностью, прежде чем выражать своё мнение.**
**Обращение к тем кто негодует из-за того что автомобиль будет таранить, для этого можно решить вопрос по ic или же если человек нарушает правила, по occ.**
Предлагаю добавить внедорожник для правоохранительных структур, а точнее Ford Police Interceptor Utility. Это будет первый внедорожник для департамента. Внедорожника очень не хватает для офицерам полиции, бываю разные случаи, при погони машин "Patriot" или других внедорожниках. Может раньше это было неактуально, но сейчас люди подкопили денюшки и купили внедорожники, почти каждый день погони за Патриотом.
Если автомобиль добавят, то сделать цену на его аренду: 120000 рублей, я думаю что вы и сами разберетесь.

![image](https://user-images.githubusercontent.com/98535300/151602556-7ed42208-b2cf-47bd-a4aa-9141567db2f1.png)
![image](https://user-images.githubusercontent.com/98535300/151602611-8e8203a9-1b57-4ae4-9860-ead2dab61992.png)

**Comment by Kradixsit** [comment]
**Я против**

Во-первых: у нас в ДПД машины бесплатные, так что их все будут брать, а во-вторых: для погони за тем же патриотом хватит и буффало.
